src/components/HomeView.jsx

```diff
@@ -60,10 +72,29 @@ function HomeView({ classes, location }) {
   let tableValue;
   let pageTitle = 'Terra Data Respository';
   let searchable = true;
+  let pageHeader = <div className={classes.title}>{pageTitle}</div>;
 
   if (tabValue === '/datasets') {
     pageTitle = 'Datasets';
     tableValue = <DatasetView searchString={searchString} />;
+    pageHeader = (
+      <div className={classes.title}>
+        <div>{pageTitle}</div>
+        <Link to="datasets/new" data-cy="create-dataset-link">
+          {Features.isEnabled('datasetSchemaCreation') && (
```

Comment:
Yeah, in this case, it doesnt actually do too much if they find their way here, but if the feature flag was protecting something that would write to the database, it could lead to some weird situations if they were pushing up malformed / incomplete data?

Possibly, we could add an `isAdmin` check too? At least the POST methods and all would still be protected by the user login information.